test/write.js

```diff
@@ -114,3 +114,18 @@ test('hex of write{Uint,Int}{8,16,32}{LE,BE} with overflow', function (t) {
   }
   t.end()
 })
+test('android issue', function (t) {
```

Comment:
Maybe name this test "Can read and write Int32BE (ref #80) or some relevant issue so its clear this will be a regression test